src/elements-experimental/seat-reservation/seat-reservation/seat-reservation.scss

```diff
@@ -173,7 +171,10 @@
 
   display: flex;
   flex-direction: column;
-  gap: 48px;
+
+  &--gap {
```

Comment:
not important for experimental state, but our code guide lines describe not to use this kind of abbreviations, but always use the full selector name. This helps to find occurrences of classes in all the repositories.